revolv/static/adjustment.scss

```diff
@@ -0,0 +1,25 @@
+@import "utils";
+@import "revolv-settings";
+
+.save-button {
+    @include revolv-hover-button($revolv-dark-blue, $revolv-dark-blue);
+    width: 97%;
```

Comment:
Again, you can make this box-sizing: border-box and width: 100% to deal with the padding: making it 97% is a bit hacky